### PR TITLE
maui-4 cleanup fiji facilities

### PR DESCRIPTION
**Pull request by avaek**
Issue: https://linear.app/bes/issue/MAUI-4/clean-up-fiji-facilities

This migration deletes the duplicate facilities specified in the above card from the database

### CODE REVIEW

**Comment by billli0** [comment]
@avaek Looks good to me! 
I saw [Emily's comment](https://linear.app/bes/issue/MAUI-4#comment-5954ccbe) requesting more updates. Maybe we can squeeze them in this PR?

**Comment by avaek** [comment]
@billli0 - great spotting! I'll include those updates in this PR too

**Comment by avaek** [comment]
@billli0 - thanks for the initial review Biao, I've updated the migration again as per the latest requirements if you get a chance to re-review it

**Review by billli0 — APPROVED** [review]
@avaek Nice work to cleanup the facilities data! Just one request to change the for loop constructs. Can give a pre approval.

File: packages/database/src/migrations/20220215235939-Maui4CleanupFijiFacilities-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      DELETE FROM entity_relation WHERE child_id = '${entityId}';
+    `);
+    await db.runSql(`
+      DELETE FROM entity WHERE id = '${entityId}';
+    `);
+    await db.runSql(`
+      DELETE FROM clinic WHERE id = '${clinicId}';
+    `);
+  });
+
```
[the remark this answers is not in the document]
Comment (avaek) [reply packages/database/src/migrations/20220215235939-Maui4CleanupFijiFacilities-modifies-data.js]: Thanks for catching that @billli0 !

I've finally got a chance to clean up the migration as per your recommendation if you get a chance to re-review it :)

**Review by billli0 — APPROVED** [review]

**Comment by edmofro** [comment]
Does this still need to happen @avaek?

**Comment by avaek** [comment]
> Does this still need to happen @avaek?

Yup, it was marked `testing failed` last week and I'll look into why tomorrow

**Pull request merged**